Oqtane.Server/Extensions/WebHostBuilderExtensions.cs

```diff
@@ -12,7 +13,7 @@ public static IWebHostBuilder ConfigureLocalizationSettings(this IWebHostBuilder
             {
                 var config = context.Configuration;
 
-                services.Configure<LocalizationOptions>(config.GetSection("Localization"));
+                services.Configure<LocalizationOptions>(config.GetSection(SettingKeys.LocalizationSection));
```

Comment:
Seems this go beyond DataDirectory ;)

Reply:
When I create a PR, it wraps in things in from subsequent commits as well.  You'll notice that is from the next commit.